apps/backend/src/libs/modules/server-application/libs/constants/constants.ts

```diff
@@ -0,0 +1,16 @@
+import { APIPath, AuthApiPath } from "~/libs/enums/enums.js";
+
+import { type WhiteRoute } from "../types/types.js";
+
+const WHITE_ROUTES: WhiteRoute[] = [
+	{
+		method: "POST" as const,
```

Comment:
doesn't it work without as const?
```suggestion
		method: "POST",
```

Reply:
@kalinkaaaa14 I believe it does, forget to remove after I've added type directly to variable.
I have a question. you asked me to add app.decorate("user", null) - do we really need it? still can't get the purpose, I mean we attach user entity to the request, not to the app.

Reply:
> doesn't it work without as const?

I believe it does. just forgot to remove after I've added the type directly to variable.